packages/jest-matchers/src/global_object.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
renaming is not hard. agreeing on the convention is :D 

i only like undercores + everything lower case (`dependency_resolver.js`) because:
- macOS is case insensitive, and case issues happen quite often ("works locally, breaks on CI", or "i renamed the file, but it's already added to git and git won't see my new name")
- no one knows when to capitalize filenames (is it when exporting classes? or objects? or functions?) and it's always a mess

using camel case tho saves 1 character everywhere you'd type an underscore, and it looks more like what you'd name a variable in JS.

it's definitely unrelated to the PR though :)